### DISCUSSION TITLE
Use event:page to track pages with id?

### DISCUSSION THREAD

**Discussion by chberge** [plausible/analytics#1935] [Anything Else]
Hi! 

Playing around with the API, and trying to get page views and unique visitors to a single page. The url would look like this: domain.app/folder/?id=unique_page_identifier  

I am trying to run 
`https://plausible.io/api/v1/stats/aggregate?site_id=domain.app&metrics=visitors,pageviews,bounce_rate,visit_duration&filters=event:page%3D%3D%2Ffolder%2F?id% unique_page_identifier`

I don't get any errors in Postman, but I also only get '0' on all results. 

What am I not missing?

**Comment by RobertJoonas** [comment]
Hi @chberge!

You get no errors because the page path is valid. It just doesn't match with any.

Plausible automatically strips all query parameters from the URL, as they can contain sensitive information. We do have a [workaround for this in our docs](https://plausible.io/docs/custom-query-params), that you can use to send query parameters in a different format. Then these will show up in the page paths and you will be able to use the `event:page` filter on it too.

Hope this helps!

**Reply by chberge** [reply]
Amazing, problem solved! Thank you!
